relay-dynamic-config/src/defaults.rs

```diff
@@ -117,6 +117,24 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 })
                 .into(),
         },
+        TagMapping {
+            metrics: vec![LazyGlob::new("d:spans/exclusive_time*@millisecond".into())],
+            tags: [
+                ("", "http.decoded_response_body_length"),
+                ("", "http.response_content_length"),
+                ("", "http.response_transfer_size"),
```

Comment:
These will have to be their own distribution metrics instead of tags. Beside the cardinality impact, having numeric values as discrete tags does not help UX, since tags are something you group by or filter on (on specific values). I will look into how we can make these separate metrics.